.github/workflows/codeql.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        branch: [ 'master', 'branch/v12', 'branch/v13', 'branch/v14' ]
+        branch: [ 'master' ]
```

Comment:
I'd recommend leaving a comment linking the RFD, to contextualize why this is only needed on master.

Reply:
Good suggestion, added